### PR TITLE
improve notifications documentation

### PR DESCRIPTION
**Pull request by LinneaAndersson**

### CODE REVIEW

File: modules/ROOT/pages/notification-details.adoc [hunk 1]
```diff
@@ -0,0 +1,1112 @@
...
+
+[source,cypher]
+----
+CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-1.0'}
+----
+
+[NOTE]
+====
+**Description of notification** +
+The `text-1.0` provider for text indexes is deprecated and will be removed in a future version.Please use `text-2.0` instead.
```
Comment (Lojjs) [inline modules/ROOT/pages/notification-details.adoc:971]:
```suggestion
The `text-1.0` provider for text indexes is deprecated and will be removed in a future version. Please use `text-2.0` instead.
```

File: modules/ROOT/pages/notification-details.adoc [hunk 2]
```diff
@@ -0,0 +1,1112 @@
...
+  MATCH (n)--(m)
+  RETURN m
+}
+RETURN *
+----
+
+[NOTE]
+====
+**Description of notification** +
+Variable in subquery is shadowing a variable with the same name from the outer scope.If you want to use that variable instead, it must be imported into the subquery using importing WITH clause. (the shadowing variable is: n)
```
Comment (Lojjs) [inline modules/ROOT/pages/notification-details.adoc:1040]:
```suggestion
Variable in subquery is shadowing a variable with the same name from the outer scope. If you want to use that variable instead, it must be imported into the subquery using importing WITH clause. (the shadowing variable is: n)
```

File: modules/ROOT/pages/notification-details.adoc [hunk 3]
```diff
@@ -0,0 +1,1112 @@
...
+
+[source,cypher]
+----
+EXPLAIN WITH $param as param RETURN param
+----
+
+[NOTE]
+====
+**Description of notification** +
+Did not supply query with enough parameters.The produced query plan will not be cached and is not executable without EXPLAIN.(Missing parameters: param)
```
Comment (Lojjs) [inline modules/ROOT/pages/notification-details.adoc:1091]:
```suggestion
Did not supply query with enough parameters.The produced query plan will not be cached and is not executable without EXPLAIN. (Missing parameters: param)
```

**Review by Lojjs — COMMENTED** [review]
Here are typos I found, also see comments in the word doc

File: modules/ROOT/pages/notification-details.adoc (hunk 2 above)
Comment (LinneaAndersson) [reply modules/ROOT/pages/notification-details.adoc:1040]: So strange, these are just copied, but it seems like something happened with the space. I'll update

**Review by Lojjs — APPROVED** [review]
I'm happy with the notification page now

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 4]
```diff
@@ -0,0 +1,1045 @@
...
+m|INFORMATION
+|Category
+m|PERFORMANCE
+|===
+
+The `CodeGenerationFailed` notification is created when it is not possible to generate a code for a query, for example, when the query is too big.
+To find more information about the specific query, see the stack trace in the _debug.log_ file.
+
+[#_hint_notifications]
+== `HINT` notifications
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:343]:
```suggestion
== `HINT` category
```

**Review by renetapopova — APPROVED** [review]
Thanks, @LinneaAndersson! All looks good for now — just one minor comment.

**Pull request merged**